routers/web/auth/auth.go

```diff
@@ -201,7 +201,8 @@ func SignInPost(ctx *context.Context) {
 
 	u, source, err := auth_service.UserSignIn(form.UserName, form.Password)
 	if err != nil {
-		if user_model.IsErrUserNotExist(err) || user_model.IsErrEmailAddressNotExist(err) {
+		if user_model.IsErrUserNotExist(err) || user_model.IsErrEmailAddressNotExist(err) ||
```

Comment:
Can't we shorten that to
```suggestion
		if errors.Is(err, util.ErrNotExist) ||
```
?

Reply:
Done.

Reply:
Doesn't look like it yet here.